python/cudf/cudf/core/dataframe.py

```diff
@@ -7209,6 +7207,14 @@ def value_counts(
         2         2            1
         6         0            1
         dtype: int64
+
+        .. pandas-compat::
+            **DataFrame.value_counts**
+
+            The returned Series will have a MultiIndex with one level per input
+            column. By default, rows that contain any NA values are omitted from
+            the result. By default, the resulting Series will be in descending
+            order so that the first element is the most frequently-occurring row.
```

Comment:
This is not a pandas compatibility note (in fact, the same note appears in the pandas docstring for this method), so can we restore to the previous state please?